nussknacker-dist/src/universal/conf/application.conf

```diff
@@ -15,7 +15,8 @@ scenarioTypes {
       classPath: ["model/genericModel.jar"]
       classPath: ${?MODEL_CLASS_PATH}
       rocksDB: {
-        checkpointDataUri: ${?FLINK_ROCKSDB_CHECKPOINT_DATA_URI}
+        enable: true
```

Comment:
is it needed? We already have it in defaultModelConfig?

Reply:
It is in `defaultModelConfig.conf` in tests but indeed - let's move it to production version of `defaultModelConfig.conf`